### ISSUE TITLE
[gatsby-plugin-facebook-analytics] Add version, cookie and xfbml config options

### ISSUE THREAD

**Issue by keithcrooks** [gatsbyjs/gatsby#14310]
## Summary

The Facebook Analytics plugin has the SDK version hard coded as `v2.12`. This should be a configurable option so that the user can choose their own preferred version without the plugin having to be updated.

### Basic example

Previous plugin config:
```javascript
plugins: [
  {
    resolve: `gatsby-plugin-facebook-analytics`,
    options: {
      appId: `YOUR_APP_ID`,
      includeInDevelopment: false,
      debug: false,
      language: `en_US`,
    },
  },
]
```

Proposed config:

```javascript
plugins: [
  {
    resolve: `gatsby-plugin-facebook-analytics`,
    options: {
      appId: `YOUR_APP_ID`,
      version: `v2.12`,
      xfbml: true,
      cookie: false,
      includeInDevelopment: false,
      debug: false,
      language: `en_US`,
    },
  },
]
```

### Motivation

The plugin is currently hard-coded to v2.12. The latest version of the Facebook JavaScript SDK is v3.3. If this change it made, it will future-proof the plugin as new versions of the SDK can be specified by providing the required version in the plugin options.

v2.12 should be the default value so as not to break backwards compatibility.

The xfbml value is also hard-coded to true. Unless the user is also using the SDK for Social Plugins, this value can be set to false to speed up page loads.

It would be a nice-to-have to also include an option for the cookie init parameter.

[Facebook JavaScript SDK initialization reference](https://developers.facebook.com/docs/javascript/reference/FB.init/v3.3)

**Comment by DSchau** [comment]
Agreed! This would be a great first contribution for somebody.

All plugin APIs have their second argument defined as pluginOptions (options passed to plugins!) so we could add this as an option and default it to 3.3.

I would specifically propose:

1. Let's take this opportunity to update to 3.3 (if it requires a major bump--that's OK!)
    - To the implementor, we're only using [`FB.AppEvents.logPageView`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-facebook-analytics/src/gatsby-browser.js#L4) which _probably_ isn't a breaking change between versions
1. You'll want to edit [the pluginOptions here](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-facebook-analytics/src/gatsby-ssr.js#L5-L10) and default version to `3.3`, and then also _use_ that version [in these lines](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-facebook-analytics/src/gatsby-ssr.js#L22)

**Comment by DSchau** [comment]
Ha - @keithcrooks you jumped ahead of me 🚀

**Comment by keithcrooks** [comment]
Yes, sorry @DSchau, I already implemented this but thought I should also add it here as a feature request with more information than in the PR.

I hope that was okay.

**Comment by DSchau** [comment]
@keithcrooks we ❤️ that you did that. It's one of our preferred ways to track on work and open PRs, actually!

Ideally - it's not _quite_ as instant as it was here (e.g. open an issue, core team replies to weigh in on request, etc.) but this seems like a really good feature/fix to get in!

**Comment by keithcrooks** [comment]
Thanks @DSchau. Did you want me to implement your suggestion of making the default v3.3?

**Comment by DSchau** [comment]
@keithcrooks I left some comments on the PR--so let's just carry that discussion over there.

In general -- I'm not a Facebook analytics user, so it will require some investigation as to whether we can make that change without a breaking change. If we can't, we should keep the PR as is. If we can--then it's OK to update to the latest!

**Comment by DSchau** [comment]
Closed by #14311
